Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM ubuntu:18.04
+FROM ubuntu:24.04
 
 COPY LICENSE README.md /
 
 RUN apt-get update && apt-get install -y \
-  clang-format-8 \
   git \
   wget \
-  && cd /usr/bin && ln -s clang-format-8 clang-format
+  && apt-get clean
+
+# Download latest .deb packages from llvm.
+# See https://apt.llvm.org/ for instructions.
+RUN apt-get update && wget https://apt.llvm.org/llvm.sh \
+    && chmod +x llvm.sh \
+    && sudo ./llvm.sh 19 \
```

Comment:
We don't have nor need `sudo` here, see [this](https://github.com/KitwareMedical/ITKRLEImage/pull/69#issuecomment-2616429184).